<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow2</class>
 <widget class="QMainWindow" name="MainWindow2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>756</width>
    <height>744</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>756</width>
    <height>744</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>756</width>
    <height>744</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Edytor Plansz</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>622</x>
      <y>15</y>
      <width>107</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>621</x>
      <y>40</y>
      <width>50</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>zapis</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>40</y>
      <width>50</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>+nowa</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>756</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPlik">
    <property name="title">
     <string>Plik</string>
    </property>
    <addaction name="actionOtw_rz_plik_z_planszami"/>
    <addaction name="actionZapisz_plik_z_planszami"/>
   </widget>
   <widget class="QMenu" name="menuEdycja">
    <property name="title">
     <string>Edycja</string>
    </property>
    <widget class="QMenu" name="menuWyczy_wszystko">
     <property name="title">
      <string>Wyczyść wszystko</string>
     </property>
     <addaction name="actionWype_nij_pustym_blokiem"/>
     <addaction name="actionWype_nij_cian"/>
     <addaction name="actionPusta_ramka"/>
     <addaction name="actionPusty_rodek"/>
     <addaction name="actionMa_a_pusta_ramka"/>
    </widget>
    <addaction name="menuWyczy_wszystko"/>
   </widget>
   <addaction name="menuPlik"/>
   <addaction name="menuEdycja"/>
  </widget>
  <action name="actionOtw_rz_plik_z_planszami">
   <property name="text">
    <string>Otwórz plik z planszami</string>
   </property>
  </action>
  <action name="actionZapisz_plik_z_planszami">
   <property name="text">
    <string>Zapisz plik z planszami</string>
   </property>
  </action>
  <action name="actionWype_nij_pustym_blokiem">
   <property name="text">
    <string>Wypełnij pustym blokiem</string>
   </property>
  </action>
  <action name="actionWype_nij_cian">
   <property name="text">
    <string>Wypełnij ścianą</string>
   </property>
  </action>
  <action name="actionPusta_ramka">
   <property name="text">
    <string>Pusta ramka</string>
   </property>
  </action>
  <action name="actionPusty_rodek">
   <property name="text">
    <string>Pusty środek</string>
   </property>
  </action>
  <action name="actionMa_a_pusta_ramka">
   <property name="text">
    <string>Mała pusta ramka</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
